nexus/test-utils/src/lib.rs

```diff
@@ -1477,7 +1477,7 @@ pub async fn start_oximeter(
     let config = oximeter_collector::Config {
         nexus_address: Some(nexus_address),
         db,
-        refresh_interval: oximeter_collector::default_refresh_interval(),
+        refresh_interval: Duration::from_secs(2),
```

Comment:
Might be worth a short comment noting why this is so short.

Reply:
Sounds good, will add!

Reply:
Done in b612396b7